etc/hexchat.profile

```diff
@@ -1,5 +1,6 @@
 # HexChat instant messaging profile
 noblacklist ${HOME}/.config/hexchat
+noblacklist /usr/lib/python2*
```

Comment:
Gotcha.

Well, the others would have been blacklisted anyways, so I think it's
best to leave it like this.

Cheers!


On 22/04/16 05:43, Reiner Herrmann wrote:
>
> In etc/hexchat.profile
> <https://github.com/netblue30/firejail/pull/466#discussion_r60643317>:
>
> > @@ -1,5 +1,6 @@
> >  # HexChat instant messaging profile
> >  noblacklist ${HOME}/.config/hexchat
> > +noblacklist /usr/lib/python2*
>
> I don't use hexchat, so I don't know what is really required.
> I just thought other scripting languages would also be required to run
> plugins/addons.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/netblue30/firejail/pull/466/files/f586c532955b81a2955aadfbfaddffe279744b54#r60643317>
>